configs/cylinder3d/metafile.yml

```
@@ -0,0 +1,29 @@
+Collections:
+  - Name: Cylinder3D
+    Metadata:
+      Training Techniques:
+        - AdamW
+      Training Resources: 4x A100 GPUs
+      Architecture:
+        - Cylinder3D
+    Paper:
+      URL: https://arxiv.org/abs/2011.10033
+      Title: 'Cylindrical and Asymmetrical 3D Convolution Networks for LiDAR Segmentation'
+    README: configs/cylinder3d/README.md
+    Code:
+      URL:
+      Version: v1.1.0rc3
```

```suggestion
      Version: v1.1.0rc4
```
This feature is only supported in the next version: v1.1.0rc4.